src/components/common/MenuTabs.tsx

```diff
@@ -31,7 +31,7 @@ const MenuTabs = ({
             </Typography>
```

Comment:
Console warning `Warning: Each child in a list should have a unique "key" prop.`. Please add a `key` prop to the `Tab` component like `key={label}`.